lib/compilation-queue.ts

```diff
@@ -42,8 +42,13 @@ const queueCompleted = new PromClient.Counter({
     help: 'Total number of jobs completed',
 });
 
+declare type Job<TaskResultType> = () => PromiseLike<TaskResultType>;
```

Comment:
What's the difference between `type` and `declare type`?

Reply:
I am not 100% sure, this was a little cargo culted from the underlying job queue I'm using.
https://stackoverflow.com/questions/43561980/what-does-declare-in-export-declare-type-xyz-mean-vs-export-type-xyz suggests it's useless, so I'll remove.

Reply:
Ah alright, I wasn't sure if there was some subtlety with `declare type` that I wasn't aware of 🙂

Reply:
Done